

root@csst-06:/home/ubuntu/laspdev# 
root@csst-06:/home/ubuntu/laspdev# python /home/ubuntu/laspdev/utility/setup_lasp.py d12
d12
ip:12.0.0.14
node:d
)2> LASPVIN no peer to subscribe case 2 
LASPVIN no peer to subscribe case 2 
lasp_peer_service:join('a@12.0.0.11').
ok
(d@12.0.0.14)3> 15:29:53.015 [error] LASPVIN received find_sub Id: "c@10.0.0.13c1" From: 'a@12.0.0.11' 
15:29:53.235 [error] LASPVIN Request forwarded~n
15:29:53.235 [error] LASPVIN received find_sub Id: "c@10.0.0.13c1" From: 'b@12.0.0.12' 
15:29:53.236 [error] LASPVIN Request forwarded~n
15:30:00.033 [error] LASPVIN received find_sub Id: "c@12.0.0.13c1" From: 'a@12.0.0.11' 
15:30:00.033 [error] LASPVIN found the peer at "2020-06-06T15:30:00.334" for ID: "c@12.0.0.13c1" ToNode: 'c@10.0.0.13' 
15:30:00.075 [error] LASPVIN received find_sub Id: "c@12.0.0.13c1" From: 'b@12.0.0.12' 
15:30:00.173 [error] LASPVIN received find_sub Id: "c@10.0.0.13c1" From: 'b@12.0.0.12' 
15:30:00.173 [error] LASPVIN Request forwarded~n
15:30:00.234 [error] LASPVIN received find_sub Id: "c@10.0.0.13c1" From: 'a@12.0.0.11' 
15:30:00.234 [error] LASPVIN Request forwarded~n
15:30:10.030 [error] LASPVIN found the peer at "2020-06-06T15:30:10.307" for ID: "c@10.0.0.13c1" ToNode: 'a@12.0.0.11' 
15:30:10.031 [error] LASPVIN found the peer at "2020-06-06T15:30:10.309" for ID: "c@12.0.0.13c1" ToNode: 'a@12.0.0.11' 
15:31:03.108 [error] LASPVIN Received delta From='b@12.0.0.12' at TimeStamp="2020-06-06T15:31:03.108" Took=41621 microseconds 
15:31:28.533 [warning] global: 'd@12.0.0.14' failed to connect to 'a@11.0.0.11'
15:31:28.533 [warning] global: 'd@12.0.0.14' failed to connect to 'a@10.0.0.11'
